searchcore/src/vespa/searchcore/proton/documentmetastore/documentmetastore.h

```diff
@@ -71,7 +74,7 @@ class DocumentMetaStore final : public DocumentMetaStoreAttribute,
     uint32_t            _shrinkLidSpaceBlockers;
     const SubDbType     _subDbType;
     bool                _trackDocumentSizes;
-    search::LidUsageStats::TimePoint _last_remove_batch;
+    documentmetastore::OperationListener::SP _op_listener;
```

Comment:
Consider using std::shared_ptr&lt;documentmetastore::OperationListener&gt;, to be consistent with forward declaration above
and not depend on corresponding include file having been included.

Reply:
Fixed